srtcore/core.cpp

```diff
@@ -9542,6 +9566,12 @@ int CUDT::processData(CUnit* in_unit)
                 // So this packet is "redundant".
                 IF_HEAVY_LOGGING(exc_type = "UNACKED");
                 adding_successful = false;
+                enterCS(m_StatsLock);
+                ++m_stats.traceRecvUniq;
+                ++m_stats.recvUniqTotal;
+                m_stats.traceBytesRecvUniq += u->m_Packet.getLength();
+                m_stats.bytesRecvUniqTotal += u->m_Packet.getLength();
+                leaveCS(m_StatsLock);
```

Comment:
It seems like this is a redundant packet, not a unique one.

Reply:
Right. This should be in the opposite block :)